soc/tsi/skyp_v1/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_SKYP_V1
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 25000000
```

Comment:
Lets make sure this value is validated by HW team.

Reply:
We will change it according to our board.